README.md

```diff
@@ -1,6 +1,160 @@
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+# 📖 [Awesome Books] <a name="about-project"></a>
+
+**[Awesome Books]** In this project, I have used a webpacl boiler plate created earlier to create a to-do-list app
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <ul>
+    <li><a href="#">HTML</a></li>
+    <li><a href="#">CSS</a></li>
+    <li><a href="#">JS</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[A webpack boiler plate with src and dist directory]**
+- **[A to-do list dynamically generated from javascript]**
+- **[html file created using the Webpack HTML plug-in]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need to:
+Have a laptop,
+Install Git,
+Install npm,
+Have a working browser,
+Have a strong internet connection,
+Install VSCode with with extentions like live server, prettier formatter etc.
+
+### Setup
+
+Install [Node Js](https://nodejs.org/en) and [NPM](https://docs.npmjs.com/cli/v6/commands/npm-install) and [Git](https://git-scm.com/downloads) on your local computer.
+
+### Install
+
+Open your commandline (if your are on windows ) or Terminal (if you are on Mac or Linux) and paste this command:
+
+```sh
+git clone https://github.com/enocol/webpack
+```
+
+then you will see the **webpack** on your system then type this:
+
+```sh
+cd webpack
+```
+
+Now you successfully installed it.
+
+### Usage
+
+on the terminal of your code editor, type:
+npm start
+
+### Run tests
+
+To run tests, run the following command:
+Use browser like chrome by clicking on the link or use live server built in VSCode
+
+### Deployment
+
+You can deploy this project using:
+
+**Github Pages**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors">Enoh Collins</a>
+
+👤 **Enoh Collins**
+
+- GitHub: [@githubhandle](https://github.com/enocol)
+- Twitter: [@twitterhandle](https://mobile.twitter.com/enocol)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/mwlite/in/enocol)
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```

Comment:
- [ ] It would be nice if you fix your twitter link because it is broken 😨 Kindly fix it to make your readme descriptive and professional 👍